skins/oasis/js/connectionType.js

```diff
@@ -0,0 +1,14 @@
+(function () {
+    'use strict';
+
+    var connection = navigator.connection || navigator.mozConnection || navigator.webkitConnection;
+
+    if (connection && connection.effectiveType) {
+        Wikia.Tracker.track({
+            action: Wikia.Tracker.ACTIONS.VIEW,
+            category: 'connection-type',
+            label: Wikia.getConnectionType(),
```

Comment:
`VM182:1 Uncaught TypeError: Wikia.getConnectionType() is not a function`

Reply:
![giphy 2](https://user-images.githubusercontent.com/2035428/32330264-da49789e-bfdf-11e7-869e-3cbe87296d6d.gif)

Reply:
Forgot to change after refactor, thanks for catch :)